docs/docs/gatsby-folder-structure.md

```diff
@@ -0,0 +1,40 @@
+## Gatsby Folder Structure:
+
+```
+/                      // Root
+|-- /.cache            // Gatsby's cache directory (created automatically)
+|-- /plugins           // (optional) To host local plugins code - link to writing plugin docs
+|-- /public            // Gatsby's output directory (created automatically)
+|-- /src               // Your project source
+    |-- /pages         // describe pages dir
+    |-- /templates     // (optional) describe templates dir
+    |-- html.js        // (optional) to customise .cache/default-html.js
+|-- /static            // (optional) Your static files that are not imported through module system.
+|-- gatsby-config.js   // describe & link to docs
+|-- gatsby-node.js     // describe & link to docs
+|-- gatsby-ssr.js      // describe & link to docs
+|-- gatsby-browser.js  // describe & link to docs
+```
+
+- **`/.cache`** This folder is created automatiacally by Gatsby. The files inside this folder is not meant for modification.
+
+- **`/plugins`** To host local plugins that are not published as an `npm` package. More info here - https://next.gatsbyjs.org/docs/plugins/
```

Comment:
Would you change all the links to the https://www.gatsbyjs.org site? That is now the main site, after we launched v2 and merged all v2 changes to the www site :)

The other change necessary is to create links using this convention [Plugin Library](https://www.gatsbyjs.org/packages/) so that the hyperlinked text is clear. See the ["use clear hyperlinks" section of the Gatsby Style Guide](https://www.gatsbyjs.org/docs/gatsby-style-guide/#use-clear-hyperlinks) for tips!